Classes/Service/CacheService.php

```diff
@@ -71,6 +80,16 @@ public function flush($force = false)
     }
 
     /**
+     * Flushes caches using the data handler. This should not be necessary any more in the future.
+     * @deprecated
+     * @internal
+     */
+    public function flushCachesWithDataHandler()
+    {
+        $this->dataHandler->clear_cacheCmd('all');
```

Comment:
We have a special customer extension, which trigger cache clearing actions to other systems. For example an multi server environment.
But if this is deprecated, we need a new handling for that.

Therefore handling of `system` is not necessary.